static/app/chartcuterie/performance.tsx

```diff
@@ -10,10 +10,38 @@ import {ChartType} from './types';
 
 export const performanceCharts: RenderDescriptor<ChartType>[] = [];
 
+type PerformanceChartData = Omit<EventBreakpointChartData, 'chartType'>;
+
 performanceCharts.push({
   key: ChartType.SLACK_PERFORMANCE_ENDPOINT_REGRESSION,
-  getOption: (data: EventBreakpointChartData) => {
-    const {chartOptions, series} = getBreakpointChartOptionsFromData(data, theme);
+  getOption: (data: PerformanceChartData) => {
```

Comment:
the actual data will be the same, we just add the chart type to the args of the method so it can pick the correct function name and possible transform function